uaclient/security.py

```diff
@@ -572,6 +574,23 @@ def fix_security_issue_id(
                     issue_id=issue_id
                 )
             raise exceptions.UserFacingError(msg)
+
+        # Check livepatch status for CVE in fixes
+        status = subprocess.run(
+            ["canonical-livepatch", "status", "--verbose", "--format=json"],
+            capture_output=True,
+            encoding="utf-8",
+        ).stdout
```

Comment:
We have a subprocess helper that includes logging - let's use that here, and we can just move on if there's an error. I think that should catch the case where canonical-livepatch isn't installed.
```suggestion
        try:
            status_stdout, _ = util.subp(
                ["canonical-livepatch", "status", "--verbose", "--format=json"],
            )
        except exceptions.ProcessExecutionError:
            pass
```

Reply:
I tried implementing `util.subp` as you have shown here, but got an error that there was no `subp` attribute of the `uaclient.util` module. Perhaps there's a different canonical repo where that is available? I searched through the rest of the `ubuntu-advantage-client` repo and didn't see any reference to a `util.subp` call, while I do see `subprocess.run()` calls.

As such, I left this as a call to `subprocess.run()` since there isn't a `util.subp` to use. Happy to pull a different branch or create this util method if desired! I did go ahead and rename the variable to `status_stdout` as you suggested here to better represent what the variable was capturing, and included a try/except block that still raises the same exception.

Reply:
Did some more digging and found the `system.subp` helper which I believe is what you were referring to before, and implemented it instead of `subprocess.run`

Reply:
Ah my bad :facepalm: we just moved that to the `system` module. Good job finding it even though I pointed you in the wrong direction!

We use `subprocess.run` in a couple places in our integration testing code, but we stick to `system.subp` in the actual cli code.